src/app/Main/Body/QueryPage/Results/Result-MainHeader.component.tsx

```diff
@@ -29,13 +25,17 @@ const tableTitleStyle = (
     display: ${isTitle ? "flex" : "none"};
     width: ${isRow ? "inherit" : `${width}px`};
     height: ${isRow ? `${height}px` : "inherit"};
-    margin: 0;
-    padding: ${isRow ? "0 0.3em" : "0.2em 0"};
+    min-height: max-content;
+    min-width: max-content;
+    margin: ${isRow ? "0.3em 0" : "0"};
+    padding: ${isRow ? "0 0.2em" : "0.2em 0"};
     justify-content: center;
     align-items: center;
     font-weight: normal;
-    background-color: ${backgroundColor};
-    color: ${primaryColor};
+    background-color: ${primaryColor};
+    color: ${onPrimaryColor};
+    writing-mode: ${isRow ? "vertical-lr" : "none"}; 

```

Comment:
i think this attribute is meant for languages with different writing alignment:
https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode
Also you already rotate the element by 180 degree so this seems obsolete.

Reply:
I don't know how else to create the look I want. 
If I rotate the font only 90 degrees, then the font overlaps the div around it.

Reply:
That's because transformations are done outside the layout rendering phase. I don't know how to do it correctly but the writing-mode + transform looks like a hack. If there is no better solution you can leave it as it is.

Reply:
In other cases, absolute positioning is used, but this makes no sense in my case. I will talk to Katja and Dominic in the new year if we really need to rotate the font.